src/NodeServices/MessageEvents.cs

```
@@ -24,6 +37,18 @@ public static void OnLogWarningMessage(string message)
         }
     }
 
+    public delegate void LogInfoMessageEventHandler(LogInfoMessageEventArgs args);
+
+    public static class LogInfoMessageEvents
+    {
+        public static event LogInfoMessageEventHandler LogInfoMessage;
+        public static void OnLogInfoMessage(string message)
```

This is the new API ProtoGeometry will call to raise info instead of warning